### ISSUE TITLE
[question] How can I pass external args to tests?

### ISSUE THREAD

**Issue by eddumelendez** [devcontainers/cli#448]
My feature depends on a secret. For that reason, I can not put it in plain text. Is there currently a way to pass an external args? In GHA, would be easy to set using `devcontainer features test -f secret-feature --args ${{ secrets.MY_TOKEN }}`

**Comment by joshspicer** [comment]
Are you looking to be able to add remoteEnv variables to a container via the command line when testing? (ie: Should these be environment variables in the running container?)

That's not something we have wired up in the test command currently, but seems like a useful feature

**Comment by joshspicer** [comment]
I could imagine a `remoteEnv` section here https://github.com/devcontainers/cli/blob/main/src/spec-node/featuresCLI/testCommandImpl.ts#L322

**Comment by eddumelendez** [comment]
yes, please that would be awesome! and probably related to this other question https://github.com/devcontainers/feature-starter/issues/41
